gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/drilling/DrillTest.java

```diff
@@ -15,45 +15,57 @@
  */
 package org.gradoop.flink.model.impl.operators.drilling;
 
+import static org.gradoop.common.GradoopTestUtils.validateIdEquality;
+
+import java.util.List;
+
+import org.apache.flink.api.java.io.LocalCollectionOutputFormat;
+import org.gradoop.common.model.impl.id.GradoopId;
+import org.gradoop.common.model.impl.pojo.GraphHead;
 import org.gradoop.flink.model.GradoopFlinkTestBase;
 import org.gradoop.flink.model.api.epgm.LogicalGraph;
 import org.gradoop.flink.model.impl.operators.drilling.drillfunctions.DrillMultiplyBy;
+import org.gradoop.flink.model.impl.operators.transformation.TransformationTest;
+import org.gradoop.flink.model.impl.functions.epgm.Id;
 import org.gradoop.flink.model.impl.operators.drilling.drillfunctions.DrillDivideBy;
 import org.gradoop.flink.util.FlinkAsciiGraphLoader;
+import org.junit.Ignore;
 import org.junit.Test;
 
+import com.google.common.collect.Lists;
+
 
 public class DrillTest extends GradoopFlinkTestBase {
 
   //----------------------------------------------------------------------------
-  // Tests for drill up
+  // Tests for roll up
   //----------------------------------------------------------------------------
 
 
   @Test(expected = NullPointerException.class)
-  public void testVertexDrillUpNoProperty() {
+  public void testVertexRollUpNoProperty() {
     FlinkAsciiGraphLoader loader = getLoaderFromString(getDrillInput());
 
     LogicalGraph input = loader.getLogicalGraphByVariable("input");
 
     LogicalGraph output = input.callForGraph(new Drill.DrillBuilder()
-      .buildDrillUp());
+      .buildRollUp());
   }
 
   @Test(expected = NullPointerException.class)
-  public void testVertexDrillUpFunctionOnly() {
+  public void testVertexRollUpFunctionOnly() {
     FlinkAsciiGraphLoader loader = getLoaderFromString(getDrillInput());
 
     LogicalGraph input = loader.getLogicalGraphByVariable("input");
 
     LogicalGraph output = input
       .callForGraph(new Drill.DrillBuilder()
         .setVertexDrillFunction(new DrillDivideBy(1000L))
-        .buildDrillUp());
+        .buildRollUp());
```

Comment:
this chain is never executed -> no assertion

Reply:
Same as above.